src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.ILLink.targets

```diff
@@ -13,6 +13,12 @@ This file contains the .NET 5-specific targets to customize ILLink
       Condition=" '$(PublishTrimmed)' == 'true' "
       AfterTargets="ComputeResolvedFilesToPublishList"
       DependsOnTargets="GetReferenceAssemblyPaths;_CreatePropertiesCache">
+    <PropertyGroup>
+      <_TrimmerDumpDependencies Condition=" '$(LinkerDumpDependencies)' == 'true' ">true</_TrimmerDumpDependencies>
+      <!-- make the output verbose to see what the linker is doing. FIXME: make dependent upon verbosity level -->
+      <_ExtraTrimmerArgs>$(_ExtraTrimmerArgs) --verbose</_ExtraTrimmerArgs>
```

Comment:
The PR should not land with --verbose

Reply:
This is unrelated to these changes. As I mentioned in https://github.com/mono/linker/issues/1030#issuecomment-606292840, illink can control the importance of its messages, but it doesn't have a way to react to MSBuild's verbosity, so that FIXME is a bit misleading - I clarified the comment. The way to get rid of this would be to make it the default, or to add an extra task argument, say `TrimmerVerbosity`.

Reply:
I'm probably missing something but setting to verbose will start printing hundreds of linker diagnostic details. Why should we do that for Android by default?

Reply:
I'm not sure why it was added exactly, maybe someone thought those diagnostics were useful. To clarify, with --verbose those diagnostics are emitted to the linker's stdout with Low importance, but this is captured by MSBuild. MSBuild's default console logger won't print Low importance messages unless you pass -v:detailed or higher. So it's up to us whether we want them to be (optionally) there. If you prefer I can also remove --verbose here, then the diagnostics won't be printed for any MSBuild verbosity.

Reply:
I'd much prefer to handle this at runtime SDK side and not by every linker consumer.

Reply:
I removed the argument and filed https://github.com/mono/linker/issues/2014 for it